tests/NHSD.BuyingCatalogue.API.IntegrationTests/Steps/ClientApplicationTypes/NativeDesktopSectionSteps.cs

```diff
@@ -25,6 +25,14 @@ public async Task ThenTheSolutionNativeDesktopHardwareRequirementsSectionContain
                 .ToString().Should().Be(value);
         }
 
+        [Then(@"the solution native-desktop native-desktop-operating-systems-description section contains (.*) with value (.*)")]
+        public async Task ThenTheSolutionNativeDesktopOperatingSystemsDescriptionSectionContainsWithValue(string token, string value)
```

Comment:
Change this to now mimic the connectivity one. Get rid of the string token parameter, as there is only one parameter

Reply:
done